app/views/provider_interface/offer/conditions/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :browser_title, title_with_error_prefix(t('.title'), @wizard.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_application_choice_respond_path(@application_choice)) %>
+
+<%= render(FlashMessageComponent.new(flash: flash)) %>
+
+<%= form_with model: @wizard, url: provider_interface_application_choice_offer_conditions_path(@application_choice), method: :post do |f| %>
+  <%= f.govuk_error_summary %>
+
+  <h1 class="govuk-heading-xl">
+    <span class="govuk-caption-xl"><%= @application_choice.application_form.full_name %></span>
+    <%= t('.title') %>
+  </h1>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= f.govuk_collection_check_boxes(:standard_conditions,
```

Comment:
There are a few style differences and missing text ("We’ll email the candidate to tell them that they’ll need to provide ID
") when comparing with the prototype, (ignoring the add another as that's out of scope) could we make them similar?

Reply:
As we decided to not tackle conditions as part of this piece of work we felt that we should ignore this, as it is included in the new piece of work.